src/pkg/services/m365/api/lists.go

```diff
@@ -85,6 +87,8 @@ var readOnlyFieldNames = keys.Set{
 
 var SkipListTemplates = keys.Set{
 	WebTemplateExtensionsListTemplateName: {},
+	DocumentLibraryListTemplateName:       {},
```

Comment:
Could you please leave a description for this set?

Also, I'm assuming that we'd eventually support backup and restore work for these list templates. Could you please open an issue if we don't have one already, and reference it in a `TODO` comment above?

Reply:
@pandeyabs I will mention an issue # over here. But does not seem like we will support backup and restore for these list templates unless it is user-reported.  

Because:
- Lists with templates `WebTemplateExtensionsListTemplateName` and `SharingLinksListTemplateName` are system generated.
- Lists with template `DocumentLibraryListTemplateName` have very different behavior than `GenericList` or `EventList`, and their backup/restore should handled via SharepointLibrary.
- And lists with `PageLibrary` template too needs to be skipped and their backup/restore should handled by SharepointPage. This template is not present in the `SkipListTemplates` KeySet as we have not encountered them in `GetLists` response.

But we have an [issue](https://github.com/alcionai/corso/issues/4932) to track the progress/updates w.r.t  `DocumentLibraryListTemplateName` and if with further deep dive it turns out that this needs to be handled w/o waiting for user report then we should implement its support.